units/console-getty.service.m4.in

```diff
@@ -9,12 +9,14 @@
 Description=Console Getty
 Documentation=man:agetty(8) man:systemd-getty-generator(8)
 After=systemd-user-sessions.service plymouth-quit-wait.service
-ConditionPathExists=/dev/console
 m4_ifdef(`HAVE_SYSV_COMPAT',
 After=rc-local.service
 )m4_dnl
 Before=getty.target
 
+# OCI containers may be run without a console
+ConditionPathExists=/dev/console
```

Comment:
I did notice the name wasn't container-specific, but I hasn't quite thought through when you'd want it.  Thanks!

This means we will want to remove RestartSec=0, for serial systems.  4bf04322b8b7ecca4f3d65cfc642d0ac16356129 "units/serial-getty@.service: use the default RestartSec".